app/assets/stylesheets/modules/_stats.styl

```diff
@@ -38,11 +38,16 @@
     &:before
       left 50%
 
-.instructor-profile-stats
-  display block
-  margin-bottom 70px
+.user_stats
   .stat-display
-    justify-content initial
+    margin 0 0 30px 0
 
-    .stat-display__stat
-      margin 0 20px 0 20px
+  #instructor-profile-stats
+    display block
+
+    .stat-display
+      justify-content initial
```

Comment:
Handles positioning of the items, here it is used to avoid the elements to space out in the horizontal space. So here I am overwriting the css properties of .stat-display for #user-prfile-stats block as there are only 2 elements in that row